c/misra/test/rules/RULE-22-8/test.c

```diff
@@ -0,0 +1,33 @@
+#include <errno.h>
```

Comment:
do we expect this to be compliant? currently is reported, I think it has something to do with way types are used in Errno.qll lib
```
void f4(void) {
  errno = 0;
  switch(errno) {
    case 0:  
      strtod("0", NULL); // COMPLIANT[FALSE_POSITIVE?]
    default:
      int i =0;
  }
}
```